### PR TITLE
Upmerge from 4.4-dev

### PR DESCRIPTION
**Pull request by laoneo**
Please hold on with the merge till we have merge all the pr's for 4.4 Alpha 2.

The last commit contains a dependency update, when needed I can revert it.

### CODE REVIEW

**Comment by richard67** [comment]
This PR will also fix issue #40770 if the dependency update is not reverted.

**Comment by richard67** [comment]
> Please hold on with the merge till we have merge all the pr's for 4.4 Alpha 2.
>
> The last commit contains a dependency update, when needed I can revert it.

@laoneo @HLeithner Please don't wait too long with making a decision about the dependency update and merging this PR. As long as there has not been made the upmerge, my tool for generating the lists of deleted files and folders will produce lots of false alarms which I have to sort out. Better merge this soon and later make again an upmerge when the few remaining PRs in the 4.4-dev branch have been merged because those few PR's have no impact on the deleted files and folders lists.

**Review by richard67 — APPROVED** [review]
Looks all right by review.

File: libraries/src/Version.php [hunk 1]
```diff
@@ -66,15 +66,15 @@ final class Version
      * @var    string
      * @since  3.8.0
      */
-    public const EXTRA_VERSION = 'alpha2-dev';
+    public const EXTRA_VERSION = 'alpha2';
```
Comment (richard67) [inline libraries/src/Version.php]: I think the changes in file "libraries/src/Version.php" should be reverted.

Comment (laoneo) [reply libraries/src/Version.php]: Good catch, that was the only file which had no conflict

**Pull request merged**